### DISCUSSION TITLE
Using Next-Auth with custom NestJS backend for OAuth?

### DISCUSSION THREAD

**Discussion by chrber04** [nextauthjs/next-auth#9579] [Help]
I'm looking to integrate next-auth into a project that utilizes a custom NestJS backend. The core of my confusion lies around the functionality of Credentials Provider and how or if it interfaces with OAuth, which is already maintained on my backend.

Does anyone have insights or experiences concerning this implementation of next-auth? Any input would be greatly appreciated. Thanks.

**Comment by emimaricic** [comment]
Looking for the same answer, did you manage to do it?
